src/parser/monk/mistweaver/modules/talents/Tier45Comparison.tsx

```diff
@@ -166,8 +166,8 @@ class Tier45Comparison extends Analyzer {
   //anaylze current play style and see how much mana they would have gained from this talent
   generateSotc() {
     const sotcBlackOutKicks = this.abilityTracker.getAbility(SPELLS.BLACKOUT_KICK_TOTM.id).damageHits || 0;
-    const manaPercentFromSotc = sotcBlackOutKicks * SPELLS.SPIRIT_OF_THE_CRANE_BUFF.manaRet;
-    const rawManaFromSotc = manaPercentFromSotc * this.manaTracker.maxResource;
+    const manaPercentFromSotc = 0.065 ;
```

Comment:
I know we want to remove .manaRet from the spell files but i don't think hard coding it here is the correct move. It should go in the constant file for mw

Reply:
as a secondary note its not .065 as that would be 6.5%. It should be .0065 as that would be the correct .65%
![image](https://user-images.githubusercontent.com/25177817/102625261-3425ab00-4113-11eb-81c6-9543eb431d82.png)